src/esp/gfx/RenderCamera.h

```diff
@@ -104,14 +104,30 @@ class RenderCamera : public MagnumCamera {
    */
   ~RenderCamera() override = default;
 
-  // Get the scene node being attached to.
+  /**
+   * @brief Get the scene node being attached to.
+   * @return the current scene node
```

Comment:
FYI the `@return` doesn't have to be there, and I don't specify it unless it actually adds some new information (here it doesn't). Same for `@param`, you either have to specify all of them or you can specify none, if the meaning is obvious.